react-app/src/components/RHF/tests/FieldArray.test.tsx

```diff
@@ -121,6 +121,16 @@ const testWrapperDependency: RHFSlotProps = {
   ],
 };
 
+vi.mock("@/api", async (importOriginal) => {
+  const actual = await importOriginal();
+  return {
+    ...(actual as object),
+    useGetCounties: vi.fn(() => {
+      return { data: [], isLoading: false, error: null };
+    }),
+  };
+});
+
```

Comment:
@13bfrancis any idea on abstracting this for the slot tests?

Reply:
Does it get used pretty frequently? This mock I mean?

Reply:
I think for now. We just accept the pain. But maybe going forward we all get together and figure out something for this. It sucks to have to continually rewrite the mocks